Gemfile

```diff
@@ -3,6 +3,6 @@ source "http://rubygems.org"
 gem 'progress', :git => 'git://github.com/djstrong/progress.git'
 gem 'slop', "~> 3.6.0"
 gem 'cycr', '~> 0.2.7'
-gem 'wiktionary'
+gem 'wiktionary', :path => '../wiktionary/'
```

Comment:
It is hard to test subprojects code dependencies. Even with github dependency I would have to push changes without testing.